stan/math/fwd/scal/fun/proj.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef STAN_MATH_FWD_SCAL_FUN_PROJ_HPP
```

Comment:
just following our (not very good) convention, can you rename this file `std_proj.hpp`?

Reply:
Yes, I will. I probably tried it in the stan namespace first and had to change it, but forgot to fix the file name.